roles/core/pxe_stack/readme.rst

```diff
@@ -161,7 +173,7 @@ To be done
 Changelog
 ^^^^^^^^^
 
-* 1.1.6: Replace atftp by fbtftp by default on RHEL/CentOS 8. Benoit Leveugle <benoit.leveugle@gmail.com>
+* 1.1.6: Add ability to handle other tftp server than atftp. Benoit Leveugle <benoit.leveugle@gmail.com>
```

Comment:
...to install other... ? Or do we handle more than the installation?

Reply:
Haha, ok, updating.